argostranslate/translate.py

```diff
@@ -264,16 +264,16 @@ def hypotheses(self, input_text, num_hypotheses=4):
         self.cache = new_cache
 
         # Construct hypotheses
-        hypotheses_to_return = []
+        hypotheses_to_return = [Hypothesis('', 0)] * num_hypotheses
```

Comment:
This looks good, but I think `[Hypothesis('', 0)] * num_hypotheses` is num_hypotheses references to the same Hypothesis object. Shouldn't matter since they should only used to compute the final Hypothesis's but potentially confusing.

Reply:
Aha.
Very good point here, I think then makes more sense to change it to list-comprehension syntax :+1: